docs/configurations.md

```diff
@@ -28,6 +28,7 @@ Configuration reference for `propeller.toml`.
 | grpc.Address                       | string          | gRPC server port to bind.                                                                                                                   |
 | grpc.PingIntervalInSec             | integer         | gRPC keepalive configuration. [Reference](https://grpc.io/docs/guides/keepalive/)                                                           |
 | grpc.PingResponseTimeoutInSec      | integer         | gRPC keepalive configuration. [Reference](https://grpc.io/docs/guides/keepalive/)                                                           |
+| grpc.EnableReflection              | bool            | If server refection should be enabled for gRPC server                                                                                       |
```

Comment:
while `bool` type is correct here, can we keep it consistent with other rows as `true/false`

Reply:
Done